dataland-frontend/tests/e2e/specs/user-api-key/GenerateApiKey.ts

```diff
@@ -0,0 +1,30 @@
+import { formatExpiryDate } from "@/utils/DateFormatUtils";
+
+describe("As a user I expect my api key will be generate correctly", () => {
+  it("successfully generate api key", () => {
+    cy.ensureLoggedIn();
+    cy.visitAndCheckAppMount("/companies");
+    cy.get("div#profile-picture-dropdown-toggle").click();
+    cy.get('[data-test="profileMenu"]').should("be.visible");
+    cy.get("div#profile-picture-dropdown-toggle").click();
+    cy.get('[data-test="profileMenu"]').should("not.exist");
```

Comment:
what is `cy.get('[data-test="profileMenu"]').should("not.exist");` good for?

Reply:
Checks if the submenu disappears when the menu button is clicked again

Reply:
ok